test/stage2/test.zig

```diff
@@ -11,19 +11,14 @@ const linux_x64 = std.zig.CrossTarget{
     .os_tag = .linux,
 };
 
-const linux_riscv64 = std.zig.CrossTarget{
-    .cpu_arch = .riscv64,
-    .os_tag = .linux,
-};
-
 pub fn addCases(ctx: *TestContext) !void {
     try @import("cbe.zig").addCases(ctx);
     try @import("spu-ii.zig").addCases(ctx);
     try @import("arm.zig").addCases(ctx);
     try @import("aarch64.zig").addCases(ctx);
     try @import("llvm.zig").addCases(ctx);
     try @import("wasm.zig").addCases(ctx);
-    try @import("darwin.zig").addCases(ctx);
+    try @import("riscv64.zig").addCases(ctx);
```

Comment:
looks like you accidentally stopped importing darwin.zig here

Reply:
Oops, my bad :sweat_smile: